openshift.spec

```diff
@@ -18,11 +20,36 @@ Source0:        https://%{import_path}/archive/%{commit}/%{name}-%{version}.tar.
 BuildRequires:  systemd
 BuildRequires:  golang >= 1.2-7
 
-Requires:       docker-io >= 1.3.2
 
 %description
 %{summary}
 
+%package master
+Summary:        Openshift Master
+Requires:       openshift = %{version}-%{release}
+Requires:       openshift-sdn
+
```

Comment:
Yeah, I think if it's an optional component it shouldn't be a hard Require. However, if the "more than 2 nodes" scenario is expected to be extremely common, it might just be worth it to keep it like this for the sake of sysadmin sanity when deploying. I'm open to either, but was just curious. Thanks for clarifying.